x-pack/plugins/threat_intelligence/README.md

```diff
@@ -6,49 +6,84 @@ Elastic Threat Intelligence makes it easy to analyze and investigate potential s
 
 The Threat Intelligence UI is displayed in Kibana Security, under the Explore section.
 
-## Quick Start
+## Development setup
 
-See the [kibana contributing guide](https://github.com/elastic/kibana/blob/main/CONTRIBUTING.md) for instructions setting up your development environment.
+### Kibana development in general
 
-Verify your node version [here](https://github.com/elastic/kibana/blob/main/.node-version).
+Best source - [internal Kibana docs](https://docs.elastic.dev/kibana-dev-docs/getting-started/welcome). If you have any issues with setting up your Kibana dev environment [#kibana](https://elastic.slack.com/archives/C0D8P2XK5) Slack channel is a good way to get help.
 
-**Run ES:**
+### Essential `kibana.yml` settings
 
-`yarn es snapshot --license trial`
+You can make a copy of `kibana.yml` file into `kibana.dev.yml` and make adjustments to the settings. External documentation on the flags available is [here](https://www.elastic.co/guide/en/kibana/current/settings.html)
 
-**Run Kibana:**
+Here is the set of recommended settings for development:
 
-> **Important:**
->
-> See here to get your `kibana.yaml` to enable the Threat Intelligence plugin.
+- You must have `xpack.securitySolution.enableExperimental: ['threatIntelligenceEnabled']` to enable Threat Intelligence capabilities built by our team (`Intelligence` page in the Security Solution)
+- It is recommended to set `server.basePath: "/kbn"` to make you local instance persist the base Kibana path. If you don't do it, the base path will be a random string every time you start Kibana. Any other value than `/kbn` will also work.
+
+## Hints
+
+Export local instance data to es_archives (will be loaded in cypress tests).
 
 ```
-yarn kbn reset && yarn kbn bootstrap
-yarn start --no-base-path
+TEST_ES_PORT=9200 node scripts/es_archiver save x-pack/test/threat_intelligence_cypress/es_archives/threat_intelligence "logs-ti*"
 ```
 
-### Performance
+### Getting Threat Intelligence feeds data into Kibana
 
-You can generate large volumes of threat indicators on demand with the following script:
+There are many ways to get data for you local development. We first focus on getting Threat Intelligence data specifically.
 
-```
-node scripts/generate_indicators.js
-```
+### Setting up filebeat threatintel integrations locally
```

Comment:
now that we moved this here, we need to update our [FAQ.md](https://github.com/elastic/kibana/blob/main/x-pack/plugins/threat_intelligence/FAQ.md) file. The last entry there points to the development-setup.mdx file that will soon be updated.

Either we move this section to the FAQ or we remove the last question in the FAQ?

Reply:
What about moving FAQ content into README.md ?

Reply:
we can! I was hoping the REAME would stay super basic, basically to get a development up and running. The bigger we make it the more confusing it _could_ be, but I'm fine with it! If the FAQ grows bigger maybe we can separate it then?

Reply:
Update: Merged FAQ section with the readme itself, and removed the note on required feature flag. Did some minor cleanups as well (mostly about loading fixtures).